bls/globals.go

```diff
@@ -11,7 +11,14 @@ func ToFr(v string) (out Fr) {
 	return
 }
 
+func init() {
+	initGlobals()
+}
+
 func initGlobals() {
+	if !EqualZero(&MODULUS) {
```

Comment:
This shouldn't happen as Go guarantees init functions are called once. Could it be that your program is linking multiple go-kzgs with different build flags?

Reply:
Right, but this is called initGlobal() and it's called from outside this file.

Reply:
Oh I see what you are saying, technically all init() functions should be called before the init functions in packages that import this one.  Hmm, let me take another look.

Reply:
Right. Every bignum implementation have mutually exclusive build tags. So only one `init` function in the `bls` package should be defined and called. Are you seeing issues in prysm?

Reply:
looks like the issue is just that MODULUS is of type Fr, and bignum_kilic Fr value is "0" !

OK perhaps I should just expose the string as the global.